Week-4/Exercise-4.2/site.js

```diff
@@ -0,0 +1,17 @@
+const Person = function () {};
+
+Person.prototype.initialize = function (name, age) {
```

Comment:
@AakashPandya Looks good to me. Nice work

Reply:
thanks @RajeshSivanesan please approve